Mathlib/CategoryTheory/Abelian/GrothendieckAxioms/FunctorCategory.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2024 Dagur Asgeirsson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Dagur Asgeirsson
+-/
+import Mathlib.CategoryTheory.Abelian.GrothendieckAxioms.Basic
+/-!
+
+# AB axioms in functor categories
+
+This file proves that, when the relevant limits and colimits exist, exactness of limits and
+colimits carries over from `A` to the functor category `C ⥤ A`
+-/
+
+namespace CategoryTheory
+
+open CategoryTheory Limits Opposite
+
+section
+
+variable {A C J : Type*} [Category A] [Category C] [Category J]
+    [HasColimitsOfShape J A] [HasExactColimitsOfShape J A] [HasFiniteLimits A]
+
+instance : HasExactColimitsOfShape J (C ⥤ A) where
```

Comment:
I wonder if this follows immediately from https://github.com/leanprover-community/mathlib4/blob/415efa61fc0b3ee799f90fe287954d6016bd1bac/Mathlib/CategoryTheory/Limits/Preserves/FunctorCategory.lean#L225

Reply:
Indeed, it does! Thanks